environment.yml

```diff
@@ -2,4 +2,4 @@ channels:
     - conda-forge
 dependencies:
     - iris=2.*
-    - python-eccodes
```

Comment:
Have to pin this as otherwise it will pick up `python-eccodes 2.12.3` which is Python 2. 
`python-eccodes 0.9.3`, confusingly, is Python 3 only

Reply:
:tired_face:  :nauseated_face: :sneezing_face:
!

Reply:
I think we should try to leave this a bit flexible.
Can we have ">0.9 <2" ??

Reply:
@lbdreyer actually I think this needs to be ">=0.9.1",
to accommodate the removal of certain workarounds, which #169 introduces.
(as on 14.x branch, this stuff was added in #138, removed in #149).